tests/test_hstrat/test_auxiliary_lib/test_alifestd_try_add_ancestor_list_col.py

```diff
@@ -0,0 +1,128 @@
+import os
+
+import pandas as pd
+import pytest
+
+from hstrat._auxiliary_lib import (
+    alifestd_convert_root_ancestor_token,
+    alifestd_make_ancestor_list_col,
+    alifestd_to_working_format,
+    alifestd_try_add_ancestor_id_col,
+    alifestd_try_add_ancestor_list_col,
+    alifestd_validate,
+)
+
+from ._impl import check_polars_implementation
+
+assets_path = os.path.join(os.path.dirname(__file__), "assets")
+
+
+alifestd_try_add_ancestor_list_col = check_polars_implementation(
```

Comment:
Instead of renaming to overwrite the original, I'd rename to end with `_` (`alifestd_try_add_ancestor_list_col_`) so there's a little hint throughout that we're using a wrapped version